arviz/data/inference_data.py

```diff
@@ -133,6 +133,18 @@ def __repr__(self):
             msg += "\n\nWarmup iterations saved ({}*).".format(WARMUP_TAG)
         return msg
 
+    def _repr_html_(self):
+        """Make html representation of object"""
```

Comment:
Would change object to `InferenceData` so this docstring can be read without having to refer to class name